### ISSUE TITLE
Adding a custom ssl fails

### ISSUE THREAD

**Issue by nq2916** [zodern/meteor-up#1064]
I have created many apps in the past with meteor & mup using the free lets-encrypt ssl and everything has worked perfectly. This new app needs to use comodossl

https://comodosslstore.com

I've attempted with the below configuration and keep getting a letsencrypt error (Error in mup setup response below). I need to somehow switch to be able to use the comodo ssl but can't find anything after searching everywhere.

```

Started TaskList: Setup Docker
[111.111.111.111] - Setup Docker
[111.111.111.111] - Setup Docker: SUCCESS

Started TaskList: Setup Meteor
[111.111.111.111] - Setup Environment
[111.111.111.111] - Setup Environment: SUCCESS

Started TaskList: Setup proxy
[111.111.111.111] - Setup Environment
[111.111.111.111] - Setup Environment: SUCCESS
[111.111.111.111] - Pushing the Startup Script
[111.111.111.111] - Pushing the Startup Script: SUCCESS
[111.111.111.111] - Pushing Nginx Config
[111.111.111.111] - Pushing Nginx Config: SUCCESS
[111.111.111.111] - Cleaning Up SSL Certificates
[111.111.111.111] - Cleaning Up SSL Certificates: SUCCESS
[111.111.111.111] - Copying SSL Certificate Bundle
[111.111.111.111] - Copying SSL Certificate Bundle: SUCCESS
[111.111.111.111] - Copying SSL Private Key
[111.111.111.111] - Copying SSL Private Key: SUCCESS
[111.111.111.111] - Setup SSL Certificates for Domains
[111.111.111.111] - Setup SSL Certificates for Domains: SUCCESS

Started TaskList: Start proxy
[111.111.111.111] - Start proxy
[111.111.111.111] x Start proxy: FAILED
    
          ------------------------------------STDERR------------------------------------
          Error response from daemon: endpoint mup-nginx-proxy not found
    Error: No such container: mup-nginx-proxy-letsencrypt
    Error response from daemon: endpoint mup-nginx-proxy-nginx-proxy not found
    docker: Error response from daemon: driver failed programming external connectivity on endpoint mup-nginx-proxy (7cd51dc5870e62e44eafd57146714d989b359f65293a9c6ffa7acb5c02a8ad6c): Bind for 0.0.0.0:80 failed: port is already allocated.
    
          ------------------------------------STDOUT------------------------------------
          mup-nginx-proxy
    Removed mup-nginx-proxy
    Removed mup-nginx-proxy-letsencrypt
    latest: Pulling from jrcs/letsencrypt-nginx-proxy-companion
    Digest: sha256:a9a558260521ada10e5d35a10f90cfd6173f5916b4e15fe5a1d3f6f650685fb2
    Status: Image is up to date for jrcs/letsencrypt-nginx-proxy-companion:latest
    Using default tag: latest
    latest: Pulling from jwilder/nginx-proxy
    Digest: sha256:619f390f49c62ece1f21dfa162fa5748e6ada15742e034fb86127e6f443b40bd
    Status: Image is up to date for jwilder/nginx-proxy:latest
    Pulled jwilder/nginx-proxy and jrcs/letsencrypt-nginx-proxy-companion
    82e8df422d022a9eeafeb997d491587149c23895fe36d764196cbfec93c56282
    
          ------------------------------------------------------------------------------
```

Mup config

```js
module.exports = {
  servers: {
    one: {
      // TODO: set host address, username, and authentication method
      host: '111.111.111.111', // Live
      username: 'root',
      pem: '~/.ssh/id_rsa'
      // password: 'server-password'
      // or neither for authenticate from ssh-agent
    }
  },

  app: {
    // TODO: change app name and path
    name: 'myapp',
    path: '../',

    servers: {
      one: {},
    },

    buildOptions: {
      serverOnly: true,
    },

    env: {
      // TODO: Change to your app's url
      // If you are using ssl, it needs to start with https://
      ROOT_URL: 'https://myapp.com',
      MONGO_URL: 'mongodb://mongodb/meteor',
      MONGO_OPLOG_URL: 'mongodb://mongodb/local',
    },

    docker: {
      // change to 'abernix/meteord:base' if your app is using Meteor 1.4 - 1.5
      image: 'abernix/meteord:node-8.4.0-base',
    },

    // Show progress bar while uploading bundle to server
    // You might need to disable it on CI servers
    enableUploadProgressBar: true
  },

  mongo: {
    version: '3.4.1',
    servers: {
      one: {}
    }
  },

  // (Optional)
  // Use the proxy to setup ssl or to route requests to the correct
  // app when there are several apps

  proxy: {
    domains: 'mydomain.com',

    ssl: {
      crt: '../pem/mypem.csr',
      key: '../pem/mypem.key',
      forceSSL: true
    }
  }
};

**Comment by nq2916** [comment]
Previous issue is fixed by:
Logging into the server and removing ssl containers that were created.
mup stop container_name
mup rm container_name

Then back on your local server inside of your project run
mup stop

Then mup setup again.

This time everything went through fine but it's still attempting to use the Let's incrypt SSL and not Comodo SSL

When I run `docker ps -a`

I get 

`CONTAINER ID        IMAGE                                    COMMAND                  CREATED              STATUS              PORTS                                      NAMES`
  `237847sjdkfj        mup-myapp:latest                         "/bin/sh -c 'bash $M…"   About a minute ago   Up About a minute   80/tcp                                     myapp`
  `2342f72d230a        mongo:3.4.1                              "/entrypoint.sh mong…"   5 minutes ago        Up 5 minutes        127.0.0.1:27017->27017/tcp                 mongodb`
  `43534c01c43d        jrcs/letsencrypt-nginx-proxy-companion   "/bin/bash /app/entr…"   5 minutes ago        Up 5 minutes                                                   mup-nginx-proxy-letsencrypt`
  `3453db960c9f        jwilder/nginx-proxy                      "/app/docker-entrypo…"   5 minutes ago        Up 5 minutes        0.0.0.0:80->80/tcp, 0.0.0.0:443->443/tcp   mup-nginx-proxy`

How do I switch to use a custom SSL?

**Comment by x5engine** [comment]
same issue on two of my apps and stuck here!!
@zodern please allow us to use our custom ssl certificates and it used to work just fine for last years!!!
thanks

**Comment by zodern** [comment]
1) Did the app use let's encrypt before?
2) Are there other apps on the server that use let's encrypt?

I have found a bug that can happen when an app previously used let's encrypt, and there was enough time between running `mup setup` and `mup deploy`, and the let's encrypt container generated the certificates between the custom certificates uploaded and the nginx container restarting. If that happened to you, you can run `mup deploy` and then run `mup setup` to fix it.

**Comment by x5engine** [comment]
@zodern I have never ever used letencrypt! 
my App use to work fine now it is down

EmbedAPI.com is down now!

I did mup setup and mup deploy many times and no luck!

I have bought a custom ssl certificate.... and used to setup it with the old versions of mup for recent year successfully but now I am really stuck now!

**Comment by zodern** [comment]
This should be fixed in mup 1.5.
